baseband/dada/base.py

```diff
@@ -267,6 +267,8 @@ def read(self, count=None, out=None):
         if out is None:
             if count is None or count < 0:
                 count = self.size - self.offset
+                if count <= 0:
```

Comment:
I think this should be `< 0` - it is fine to return an empty array for `count == 0` (which will automatically happen).

Reply:
Haha, this is because I didn't realize doing so is self-consistent with `fh.read(0)`, which does return `array([])`.

Reply:
Yes, `fh.read()` gives an empty result even if one is beyond the end of the file...